data/map systems.txt

```diff
@@ -14312,6 +14504,70 @@ system Fscher
 		distance 15894
 		period 10098.0
 
+system Fumalasamakah
```

Comment:
@Ebreus

Reply:
![image](https://github.com/endless-sky/endless-sky/assets/18634983/5dae0a1c-05b8-4d41-be6e-48a5b77f438e)

Reply:
Ah, okay!  I didn't catch this comment at first because it's image-only so it gets lost in the feed.  That solves that then.  An o5v is probably a good stand in for a b6ve, too :-)

Reply:
Under the idea that we want to keep things closer to the IRL descriptions as you mentioned, I've changed it from an o5 to a b5. The o5 was purple-hued, when the above states it as blue-hued.

Reply:
Sure.  Either way; I was thinking the purple could be a sort of approximation of the extra emission lines, but legit b5 works, too.

Reply:
Should the name of the system be changed to match the real life system, while we still can change it?